src/redux/thunks/previewHelmValuesFile.ts

```diff
@@ -68,8 +66,8 @@ export const previewHelmValuesFile = createAsyncThunk<
 
       if (result.stdout) {
         return createPreviewResult(
-          String(k8sVersion),
-          String(userDataDir),
+          projectConfig.k8sVersion || PREDEFINED_K8S_VERSION,
```

Comment:
The `projectConfig.k8sVersion || PREDEFINED_K8S_VERSION` is used once again. If we will want to refactor this it will be easier to have a single point of entry